docs/Contributing/product-groups/orchestration/understanding-host-vitals.md

```diff
@@ -1085,7 +1087,8 @@ SELECT
   browser_type AS extension_for,
   'chrome_extensions' AS source,
   '' AS vendor,
-  path AS installed_path
+  path AS installed_path,
+	'' as upgrade_code
```

Comment:
Super nit pick but the indentation doesn't match in these statements

Reply:
This is what's automatically generated from `make generate-doc`, so manually changing it causes CI failure